chart/epinio/values.yaml

```diff
@@ -60,8 +60,8 @@ s3:
 api:
   # Default user username
   username: admin
-  # Default user password
-  password: password
+  # Default user password (bcrypt hash for 'password')
+  password: "$2a$10$6bCi5NMstMK781In7JGiL.B44pgoplUb330FQvm6mVXMppbXBPiXS"
```

Comment:
I feel the name of this value should change to something like `encryptedPassword` or even `passwordBcrypt`. Another option would be to let the password be plain text in values and we do the encryption in helm (if supported). But in that case, the plain-text password will float around in values.yaml files and command line histories, so let's not.

Reply:
Renamed to `passwordBcrypt` just to make it more explicit the algorithm used